kexec/kexec.nix

```diff
@@ -39,10 +39,12 @@
     mkdir -p /mnt-root/root/.ssh/
     cp /authorized_keys /mnt-root/root/.ssh/
   '';
-  system.build.kexec_tarball = pkgs.callPackage <nixpkgs/nixos/lib/make-system-tarball.nix> {
+  system.build.kexec_tarball = pkgs.callPackage (pkgs.path + "/nixos/lib/make-system-tarball.nix") {
     storeContents = [
       { object = config.system.build.kexec_script; symlink = "/kexec_nixos"; }
     ];
-    contents = [];
+    contents = [
+      { source = image; target = image; }
```

Comment:
We would also need this change in nixos-generators: @Lassulus

Reply:
maybe we should duplicate the code in nixos-generators at some point? but not sure